docker-centos/Dockerfile

```diff
@@ -16,6 +16,10 @@ rpm -V docker-rhel-push-plugin && \
 mkdir -p /exports/hostfs/etc/docker && \
 yum clean all
 
+# Move the /etc/sysconfig/docker out of the way as daemon.json is the proper config location
```

Comment:
do we need to do this?  I am afraid we will break the existing configuration for docker on the host, in case the user wants to still keep it.  Is it enough to drop it from the .service file?